src/main/kotlin/com/vanniktech/rxriddles/Riddle35.kt

```diff
@@ -0,0 +1,15 @@
+package com.vanniktech.rxriddles
+
+import io.reactivex.Maybe
+import io.reactivex.Single
+
+object Riddle35 {
+  /**
+   * Return a Single that emits the value from the primary source if present, else emits from secondary.
```

Comment:
`Return a Single that emits the value from the [first] source if present, otherwise emit from the [second] source.`